src/lib/updates.js

```diff
@@ -47,11 +40,17 @@ export const createUpdate = async (files = [], channel, ts, user, text) => {
         videoPlaybackIds.push(publicUrl.muxPlaybackId);
       }
     }),
-  ]).then((values) => {
-    if (values[1] === "error") return "error";
-  });
+  ])
+
+  if ((attachments.length + videos.length) === 0) {
+    await Promise.all([
+      react("remove", channel, ts, "beachball"),
+      react("add", channel, ts, "x"),
+    ]);
+    metrics.increment("errors.file_upload", 1);
+    return "error";
+  }
```

Comment:
Let's put the deleting logic in here. And let's make all error messages ephemeral using:

```
postEphemeral(channel, t("messages.errors.heic"), user),
```